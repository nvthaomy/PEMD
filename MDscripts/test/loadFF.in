source leaprc.gaff2
source leaprc.ff99
source leaprc.water.opc
loadOFF PAAP9N9c5.lib
loadOFF PAAD9N9c4.lib
loadOFF PAHP1N11c5.lib
loadOFF PAHD1N11c6.lib

x=loadpdb mixture.pdb
addions x Na+ 0
addions x Cl- 0
setbox x vdw 1
saveamberparm x mixture.parm7 mixture.crd
savepdb x mixture.pdb
quit