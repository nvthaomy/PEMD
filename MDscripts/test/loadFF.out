-I: Adding /home/mnguyen/miniconda3/dat/leap/prep to search path.
-I: Adding /home/mnguyen/miniconda3/dat/leap/lib to search path.
-I: Adding /home/mnguyen/miniconda3/dat/leap/parm to search path.
-I: Adding /home/mnguyen/miniconda3/dat/leap/cmd to search path.
-s: Ignoring startup file: leaprc
-f: Source loadFF.in.

Welcome to LEaP!
Sourcing: ./loadFF.in
----- Source: /home/mnguyen/miniconda3/dat/leap/cmd/leaprc.gaff2
----- Source of /home/mnguyen/miniconda3/dat/leap/cmd/leaprc.gaff2 done
Log file: ./leap.log
Loading parameters: /home/mnguyen/miniconda3/dat/leap/parm/gaff2.dat
Reading title:
AMBER General Force Field for organic molecules (Version 2.11, May 2016)
----- Source: /home/mnguyen/miniconda3/dat/leap/cmd/leaprc.ff99
----- Source of /home/mnguyen/miniconda3/dat/leap/cmd/leaprc.ff99 done
Log file: ./leap.log
Loading parameters: /home/mnguyen/miniconda3/dat/leap/parm/parm99.dat
Reading title:
PARM99 for DNA,RNA,AA, organic molecules, TIP3P wat. Polariz.& LP incl.02/04/99
Loading library: /home/mnguyen/miniconda3/dat/leap/lib/all_nucleic94.lib
Loading library: /home/mnguyen/miniconda3/dat/leap/lib/all_amino94.lib
Loading library: /home/mnguyen/miniconda3/dat/leap/lib/all_aminoct94.lib
Loading library: /home/mnguyen/miniconda3/dat/leap/lib/all_aminont94.lib
Loading library: /home/mnguyen/miniconda3/dat/leap/lib/ions94.lib
Loading library: /home/mnguyen/miniconda3/dat/leap/lib/solvents.lib
----- Source: /home/mnguyen/miniconda3/dat/leap/cmd/leaprc.water.opc
----- Source of /home/mnguyen/miniconda3/dat/leap/cmd/leaprc.water.opc done
Loading library: /home/mnguyen/miniconda3/dat/leap/lib/atomic_ions.lib
Loading library: /home/mnguyen/miniconda3/dat/leap/lib/solvents.lib
Loading parameters: /home/mnguyen/miniconda3/dat/leap/parm/frcmod.opc
Reading force field modification type file (frcmod)
Reading title:
Parameters for OPC water (JPCL, 2014, 5 (21), pp 3863â€“3871) 
Loading parameters: /home/mnguyen/miniconda3/dat/leap/parm/frcmod.ionsjc_tip4pew
Reading force field modification type file (frcmod)
Reading title:
Monovalent ion parameters for Ewald and TIP4P/EW water from Joung & Cheatham JPCB (2008)
Loading parameters: /home/mnguyen/miniconda3/dat/leap/parm/frcmod.ions234lm_126_tip4pew
Reading force field modification type file (frcmod)
Reading title:
Li/Merz ion parameters of divalent to tetravalent ions for TIP4P/EW water model (12-6 normal usage set)
Loading library: ./PAAP9N9c5.lib
Loading library: ./PAAD9N9c4.lib
Loading library: ./PAHP1N11c5.lib
Loading library: ./PAHD1N11c6.lib
Loading PDB file: ./mixture.pdb
 (starting new molecule for chain B)
 (starting new molecule for chain C)
 (starting new molecule for chain D)
 (starting new molecule for chain E)
  total atoms in file: 644

/home/mnguyen/miniconda3/bin/teLeap: Warning!
addIons: 1st Ion & target unit have charges of the same sign:
     unit charge = 7e-05; ion1 charge = 1;
     can't neutralize.
 0.000070 0 1 0
0 Cl- ion required to neutralize.
Adding 0 counter ions to "x" using 1A grid
Box dimensions:  23.520000 23.520000 23.520000
Checking Unit.

/home/mnguyen/miniconda3/bin/teLeap: Note.
Ignoring the warnings from Unit Checking.

Building topology.
Building atom parameters.
Building bond parameters.
Building angle parameters.
Building proper torsion parameters.
Building improper torsion parameters.
 total 10 improper torsions applied
Building H-Bond parameters.
Incorporating Non-Bonded adjustments.
Not Marking per-residue atom chain types.
Marking per-residue atom chain types.
  (Residues lacking connect0/connect1 - 
   these don't have chain types marked:

	res	total affected

	AHP	1
	ATD	1
	NHD	1
	NTP	1
	WAT	100
  )
 (no restraints)
Writing pdb file: mixture.pdb
   printing CRYST1 record to PDB file with box info
	Quit

Exiting LEaP: Errors = 0; Warnings = 1; Notes = 1.
